File: implementations/node/sdk/src/contracts/control/execution-control-action.ts
export const AI_SDK_EXECUTION_CONTROL_ACTIONS = [
  "None",
  "Pause",
  "Resume",
  "Cancel",
  "WaitForInput",
  "SubmitInput",
] as const;

export type AiSdkExecutionControlAction =
  (typeof AI_SDK_EXECUTION_CONTROL_ACTIONS)[number];

File: implementations/python/sdk/src/multiplexed_ai_sdk/contracts/control/execution_control_action.py
from enum import Enum


class AiSdkExecutionControlAction(str, Enum):
    NONE = "None"
    PAUSE = "Pause"
    RESUME = "Resume"
    CANCEL = "Cancel"
    WAIT_FOR_INPUT = "WaitForInput"
    SUBMIT_INPUT = "SubmitInput"
